eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -53,14 +53,6 @@ jobs:
           ArtifactName: 'packages'
 
     steps:
-      - task: PowerShell@2
```

Comment:
Does Dozie's changes still depend on this?

Reply:
Thanks for the catch I do believe it does. This looks like a bad merge I will fix it.